pkgs/by-name/si/sieve/package.nix

```
@@ -0,0 +1,109 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  electron,
+  copyDesktopItems,
+  makeDesktopItem,
+  stdenv,
+  xcbuild,
+}:
+
+buildNpmPackage rec {
+  pname = "sieve";
+  version = "0.6.1-unstable-2024-07-25";
+
+  src = fetchFromGitHub {
+    owner = "thsmi";
+    repo = pname;
```

It's better to use a string literal here (see https://github.com/NixOS/nixpkgs/issues/277994)

```suggestion
    repo = "sieve";
```